apps/user-portal/src/actions/consent-management.ts

```diff
@@ -311,132 +311,132 @@ export const updateRevokedClaimIds = (claimIds: number[]): UpdateRevokedClaimIds
  *
  * @type {string}
  */
-export const SHOW_CONSENTS_EDIT_VIEW = "SHOW_CONSENTS_EDIT_VIEW";
+export const SHOW_CONSENT_EDIT_VIEW = "SHOW_CONSENT_EDIT_VIEW";
```

Comment:
Move actionTypes to a separate file. May be we can create a `types` folder inside `actions` folder.

Reply:
A separate folder was created inside actions directory and corresponding ```.ts``` files were created and the action types were moved in to those files.